session/session.go

```diff
@@ -55,8 +56,12 @@ func createSession(ctx context.Context, params types.ExecCreateParams, gpuType s
 }
 
 func createSessionFromConfigGPUTypes(ctx context.Context, params types.ExecCreateParams) (*types.Exec, error) {
-	gpuTypesFromConfig := gpuTypesFromConfig()
-	var err error
+	gpuTypesFromConfig, err := gpuTypesFromConfig()
+	if err != nil {
+		ui.Errorf(err.Error())
```

Comment:
This again exposes the internal error below. i.e. the user doesn't know what `GPU type config should be provided or default, never nil` means. We probably don't want to handle an error here and instead do it in the `gpuTypesFromConfig` function. 

We will eventually need to refactor the error handling. A good convention might be to nevel `os.Exit` or print `ui.*` from a package that is not `cmd`. i.e. the `cmd` package handles all user interface things while sub return meaningful errors. 

For now, let's just fix this error to not expose internal semantics.

Reply:
I would probably lean towards doing a combination of this and using a function to handle errors on the CLI that only prints errors that are intended to be directed at the user.